serverless/knative_eventing/serverless-channels.adoc

```diff
@@ -13,14 +13,8 @@ After events have been sent to a channel, these events can be sent to multiple K
 The default configuration for channel instances is defined in the `default-ch-webhook` ConfigMap. However, developers can still create their own channels directly by instantiating a supported channel object.
 
 == Supported channel types
-Currently, {ServerlessProductName} only supports the use of InMemoryChannel channels as part of the Knative Eventing Technology Preview.
-
-The following are limitations of InMemoryChannel channels:
 
-* No event persistence is available. If a Pod goes down, events on that Pod will be lost.
-* InMemoryChannel channels do not implement event ordering, so two events that are received in the channel at the same time may be delivered to a subscriber in any order.
-* If a subscriber rejects an event, there are no redelivery attempts. Instead, the rejected event is sent to a _dead letter sink_ if this sink exists, or is otherwise dropped.
-
-// TODO: Add link for dead letter sink docs once these are added.
+Currently, {ServerlessProductName} only supports the use of InMemoryChannel channels as part of the Knative Eventing Technology Preview.
```

Comment:
s/channels/Channels to match the rest of the docs.

Reply:
I've actually changed it all to lower case now until we figure out what should or shouldn't be capitalized :) I am only using "Channel" now where this specifically refers to the `Channel` object.